src/components/Comment.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from 'react';
+import { Avatar } from '@mui/material';
+
+
+interface CommentProps {
+    avatar: string, 
+    username: string,
+    text: string,
+    creationDate: string
+}
+
+export function Comment(props: any) {
+    const {avatar, username, text, creationDate} = props;
```

Comment:
You can destructure props directly in function like this: 
`Component({ avatar, username, text, creationDate })`